tests/e2e/verfication_post_day1_test.go

```diff
@@ -11,13 +11,15 @@ import (
 	CI "github.com/terraform-redhat/terraform-provider-rhcs/tests/ci"
 	CMS "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/cms"
 	CON "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	EXE "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
 	H "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/helper"
 )
 
 var _ = Describe("TF Test", func() {
 	Describe("Verfication/Post day 1 tests", func() {
 		var err error
 		var profile *CI.Profile
+		var vpc *EXE.VPCService
```

Comment:
You just clarify the attribute vpc. You didn't initialize it, that why you got nil pointer.
Add line `vpcService = EXE.NewVPCService()` to your beforeeach if you need it for every test case. Otherwise, put to the required case